tests/unit/test_signature.cpp

```diff
@@ -35,27 +36,41 @@ class SignatureTest : public ::testing::Test
     void SetUp() override;
 
 protected:
-    std::unique_ptr<AsymmetricKeypair> _keyPair;
+    std::unique_ptr<AsymmetricKeypair> _keyPairRsa;
+    std::unique_ptr<AsymmetricKeypair> _keyPairEcc;
 
-    std::unique_ptr<AsymmetricPublicKey> _root1_pubkey;
-    std::unique_ptr<X509Certificate> _root1_cert;
+    std::unique_ptr<AsymmetricPublicKey> _root1_RsaPubkey;
+    std::unique_ptr<X509Certificate> _root1_RsaCert;
 
     static const std::string _testMessage;
 
-    // The SHA256 digest of the test message.
+    /* The SHA256 digest of the test message. */
     static const std::vector<uint8_t> _testMessageDigestSHA256;
-    // The SHA512 digest of the test message.
+    /* The SHA512 digest of the test message. */
     static const std::vector<uint8_t> _testMessageDigestSHA512;
-
-    // Two valid keys belonging to the same key pair.
-    static const std::string _validPublicKey;
-    static const std::string _validPrivateKey;
-
-    // The signatures of the hashed test message with the previously declared private key
+    /* The SHA1 digest of the test message. */
+    static const std::vector<uint8_t> _testMessageDigestSHA1;
+
+    /* Valid keys belonging to the same key pair. */
+    static const std::string _validRsaPublicKey;
+    static const std::string _validEccPublicKey;
```

Comment:
Maybe we should add a small comment with the information of what curve was chosen to generate the keys.